cmd/lakectl/cmd/local.go

```diff
@@ -67,7 +67,9 @@ func localDiff(ctx context.Context, client apigen.ClientWithResponsesInterface,
 		return local.ListRemote(ctx, client, remote, currentRemoteState)
 	})
 
-	changes, err := local.DiffLocalWithHead(currentRemoteState, path)
+	includeDirs := cfg.Experimental.Local.UnixPerm.Enabled
```

Comment:
Separately, the name "UnixPerm" is extremely confusing and should be replace with e.g. "POSIXPermissions".

AFAIU these are a particular subset of POSIX permissions.  A fair proportion of our users (those on Linux; in future, those on Windows) will _not_ be using this with Unix.  Meanwhile Unix supports and supported a much wider array of permissions than POSIX.

Reply:
You can change that